cpp/include/cudf/strings/detail/split_utils.cuh

```diff
@@ -13,16 +13,19 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+#pragma once
```

Comment:
```suggestion

#pragma once
```

Reply:
We're highly inconsistent about whether a newline is needed after the license. I'd say this is not something worth worrying about in the future.
```bash
~/code/cudf $ rg "\*/\n#pragma once" --multiline --count | wc -l
186
~/code/cudf $ rg "\n\n#pragma once" --multiline --count | wc -l
240
```